src/components/navBar/navBar.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
```

Comment:
React Typescript docs recommend importing React like so:
import * as React from "react";
although there is nothing inherently wrong with importing normally, I would stick with the futureproof way.
**Reference** - https://react-typescript-cheatsheet.netlify.app/docs/basic/setup#import-react